fbpic/particles/elementary_process/cuda_numba_utils.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2017, FBPIC contributors
```

Comment:
This is the generic file that contains methods that can be used by ionization, but also other elementary processes (e.g. Compton)